django-stubs/contrib/admin/options.pyi

```diff
@@ -76,19 +76,19 @@ _ListFilterT: TypeAlias = (
 _ModelT = TypeVar("_ModelT", bound=Model)
 
 class BaseModelAdmin(Generic[_ModelT]):
-    autocomplete_fields: Sequence[str]
-    raw_id_fields: Sequence[str]
+    autocomplete_fields: _ListOrTuple[str]
+    raw_id_fields: _ListOrTuple[str]
     fields: _FieldGroups | None
-    exclude: Sequence[str] | None
+    exclude: _ListOrTuple[str] | None
```

Comment:
Should we update the getter methods below as well, e.g. `get_exclude(...) -> Sequence[str]` to `_ListOrTuple`?

Reply:
Hm, I'm not totally sure but it could probably be reasonable to align? I mean, per default these methods just do `return self.<attribute>`

Reply:
Yeah my preference would be to update methods as well.

Reply:
Yeah, I'll push a new PR with fixes very soon. Trying to check their alignment just now.

Reply:
See #1833